skycultures/tikuna/description.en.utf8

```diff
@@ -0,0 +1,39 @@
+<h1 align="center"> The Skyscape: Tikuna Constellations and Seasonality Tikuna</h1>
+
+<h3 align="justify">Priscila Faulhaber<br>MAST (Museu de Astronomia e Ciências Afins (Brasil)</br></h3>
+
+<p align="justify">The movements of the Tikuna constellations correspond to an interpretation of the seasonality of rainfall, as the Tikuna narratives suggest. These constellations are related to myths referencing the ascension and fall of Wücütcha (a heavenly beast related to the Jaguar Clan), Coyatchicüra (Caiman’s Jaw) and Baweta (Tracajá, the Aquatic Turtle Collective), the Anteater’s fight with the Jaguar, and the Way of the Tapir.</p>
+<p align="justify">According to the testimony of the Tikuna indigenous people, Baweta appears in the eastern sky towards the end of November, at nightfall. After Baweta rises (Pleiades Star Cluster), Caiman’s Jaw rises. Wücütcha’s leg corresponds to a portion of The Orion Constellation; and it begins to disappear from the western sky around the 20th of May, at nightfall. About one month earlier, the Caiman’s Jaw, corresponding to The Hyades Star Cluster in Taurus, also disappears to the West.</p>
+<p align="justify">The Tikuna indigenous people have inhabited the region of the Upper Solimões river for the past 2,000 years. They currently occupy a territory where the political borders of three countries converge: Brazil, Colombia, and Peru. This is the reason why the Tikuna recognize themselves as a differentiated indigenous group, while also identifying with the national identities related to these three countries.</p>
+<p align="justify">Tikuna territory encompasses a region 600 kilometers long, running west to east from the Chimbote region in Peru—along the emptiness of the Amazonas-Solimões valley, in the so-called Trapézio Amazônico (Colombian Amazonian Trapezoid)—to the Barreira da Missão region, in the municipality of Tefé, in the Brazilian state of Amazonas (See The Ticuna Atlas, Figure I).<p>
+<h5 align="center"><img src="Atlas_das_terras_Ticuna.png" width="753" height="453" /><br/>
+<em>Figura 1: The Ticuna Atlas, Figure I<cite id="cit1a"><a href="#cit1"><B>¹</B></a></cite> </em></h5>
```

Comment:
Ticuna or Tikuna?

Reply:
hello here if you use both but I'm going to standardize for tikuna I'm already providing the changes thank you